Source/WebCore/rendering/svg/SVGResources.cpp

```diff
@@ -307,22 +307,38 @@ std::unique_ptr<SVGResources> SVGResources::buildCachedResources(const RenderEle
     return foundResources;
 }
 
-void SVGResources::layoutDifferentRootIfNeeded(const LegacyRenderSVGRoot* svgRoot)
+void SVGResources::layoutDifferentRootIfNeeded(const RenderElement& resourcesClient)
 {
+    const LegacyRenderSVGRoot* clientSVGRoot = nullptr;
+
     auto layoutDifferentRootIfNeeded = [&](RenderElement* container) {
```

Comment:
Should not the type be `LegacyRenderSVGResourceMarker` instead of `RenderElement`? And should not the name be `resourceContainer` or `resource` instead of the generic name `container`?